spec/spec_helper.rb

```diff
@@ -9,6 +9,7 @@
 
 browser = Billy::Browsers::Watir.new :phantomjs
 Capybara.app = Rack::Directory.new(File.expand_path('../../examples', __FILE__))
+Capybara.server = :webrick
```

Comment:
This is a fix for Travis CI, where some versions of Ruby / Capybara use `puma`:

![image](https://user-images.githubusercontent.com/3383704/42097543-626cb846-7b86-11e8-97b8-5ca622a210f7.png)